city_scrapers/spiders/chi_ssa_60.py

```diff
@@ -0,0 +1,129 @@
+import html
+import json
+from datetime import datetime
+
+from city_scrapers_core.constants import ADVISORY_COMMITTEE, COMMISSION, COMMITTEE, NOT_CLASSIFIED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from dateutil.relativedelta import relativedelta
+from scrapy import Request
+
+
+class ChiSsa60Spider(CityScrapersSpider):
+    name = "chi_ssa_60"
+    agency = "Chicago Special Service Area #60 Albany Park"
+    timezone = "America/Chicago"
+    months_backward = 10
+    months_forward = 1
+
+    start_urls = None  # Generated by `start_requests`
+    custom_settings = {'ROBOTSTXT_OBEY': False}
+
+    def start_requests(self):
+        requests = []
+        for i in range(-self.months_backward, self.months_forward + 1):
+            dt = datetime.strftime(datetime.now().date() + relativedelta(months=i), "%Y-%m")
+            requests.append(
+                Request(
+                    "https://northrivercommission.org/events/{}/".format(dt), callback=self.parse
+                )
+            )
+        return requests
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        event_list = list()
+        for item in response.xpath("//script[@type='application/ld+json']/text()"):
+            event_list = json.loads(self._clean(item.get()))
+
+        for item in event_list:
+            title = self._parse_title(item)
+            if title is None:
+                continue
+
+            meeting = Meeting(
+                title=title,
+                description=self._parse_description(item),
+                classification=self._parse_classification(title),
+                start=self._parse_start(item),
+                end=self._parse_end(item),
+                all_day=False,  # Found no events of interest scheduled for all day
+                time_notes="",
+                location=self._parse_location(item),
+                links=list(),
+                source=item["url"],
+            )
+
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+
+            yield meeting
+
+    def _parse_title(self, item):
+        """Parse or generate meeting title."""
+        allowed = ['meeting', 'committee', 'advisory council']
+        if not any([a in item['name'].lower() for a in allowed]):  # Not a meeting type we want
+            return None
+        return item['name']
+
+    def _parse_description(self, item):
+        """Parse or generate meeting description."""
+        desc = html.unescape(item["description"])
```

Comment:
I think this is good to leave in. Just to be safe here in case they add any more HTML in, it might be easier to create a `Selector` from the description and extract all the text inside of it like we do here: https://github.com/City-Bureau/city-scrapers/blob/20a12ba5d76186cba65b45f7f764f02393d4a991/city_scrapers/spiders/cook_landbank.py#L59

I'm also seeing some miscellaneous whitespace characters like `\xa0` in the output, so it might make sense to replace all whitespace characters with a single space with `re.sub(r"\s+", " ", string).strip()`

Reply:
Hmm, yeah my approach definitely felt clunky. This sounds a lot better. If I'm understanding you properly though, that would also depend on every description being contained within the `<p></p>` tags right? I don't remember off hand if they all were, but I'll take a look.

Reply:
It should work without any tags if you just check pull everything out with:

```python
" ".join(item.css("*::text").extract())
```

Reply:
Hmm, that worked really well. I'm going to have to dive into the CSS selectors someday, I haven't really ever used them. Thanks!

Reply:
I'm partially just more used to them from doing front-end work, but I think they're a lot easier to use and I'd definitely recommend it!